Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml

```diff
@@ -482,3 +482,13 @@
   - type: Icon
     sprite: Structures/Storage/Crates/stone_grave.rsi
     state: base
+    
+- type: entity
+  parent: CrateBaseWeldable
+  id: CrateSyndicate
```

Comment:
This could use a localized entity name and description, like how the `CrateSyndicateSuperSurplusBundle` has.

```
ent-CrateSyndicateSuperSurplusBundle = Syndicate super surplus crate.
    .desc = Contains 125 telecrystals worth of completely random Syndicate items.
```